src/app/public/modules/toolbar/toolbar-view-actions.component.ts

```diff
@@ -3,6 +3,11 @@ import {
   Component
 } from '@angular/core';
 
+/**
+ * Adds a section on the right side of the toolbar for items that substantially alter
+ * the view of the content container. This includes simple filters and view switchers.
+ * For best practices on the order of toolbar items and item placement, see UX guidelines.
```

Comment:
I'd change the opening clause to `For best practices on the order and placement of toolbar items,`

Also, we should either change `UX guidelines` to `the Design tab` or get rid of this sentence altogether. (I don't think we can link to a section on an other tab as of right now.)